lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -936,6 +937,16 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self._rm_client = get_resource_management_client(
             self.credential, self.subscription_id, self.cloud
         )
+        if self.runbook.admin_private_key_file:
```

Comment:
It should be below the generating. So the generating doesn't need to return public key data together.